### PR TITLE
refactor: remove everything not token-related, rename tokens to coins

### LINKED ISSUE #1: Remove account management
https://github.com/Hydractify/forgottenserver/ has been archived due to us using a more update project, which, requires to an extent an account manager to be in place, making the account management in guardian of tales obsolete.

### LINKED ISSUE #2: Replace tokens with tibia coins
With https://github.com/Hydractify/forgottenserver/ being archived, we moved to a more updated project which uses the `12.60` protocol, having a built-in shop within the game that uses coins as it's currency.

### PR DESCRIPTION
**Pull request by SpaceEEC**
This PR removes everything account-management related and renames tokens to coins.
This PR also makes use of Discord's new slash commands (which are in beta).

~~There currently does not seem to be a way to get the account from a discord user id.
This needs to be taken care of before merging this PR.~~
It's now possible to link your Discord account through our myaac fork.

Resolves #1
Resolves #2

# Before Merging
- [x] Address amount of coins being granted in $INTERVAL
- [x] Address how we determine what account belongs to the what user

### CODE REVIEW

**Review by wizardlink — COMMENTED** [review]
Looks good to me, but, have you checked if the amount of coins a user can get per message is in line with the new shop?

30 days of premium is 300 coins, not 10000.

**Comment by SpaceEEC** [comment]
No, I did not touch that part.
The behavior is still the same: 1 coin per not-command-message if the last one granting coins was longer than a second ago.

**Comment by wizardlink** [comment]
> No, I did not touch that part.
> The behavior is still the same: 1 coin per not-command-message if the last one granting coins was longer than a second ago.

Alright, I'll revise that later on then.

**Review by wizardlink — DISMISSED** [review]

**Comment by wizardlink** [comment]
Well, PR was put aside but, I believe we should re-implement the `gamble` command and update the fields since they have changed. See: https://github.com/Hydractify/otservbr-global/commit/35b42b110c98ec9582f08d01832cde015fd7a78c

File: src/structures/Client.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ declare module "discord-akairo"
 export class Client extends AkairoClient
 {
 	public readonly commandHandler: CommandHandler;
-	public readonly inhibitorHandler: InhibitorHandler;
+	// public readonly inhibitorHandler: InhibitorHandler;
```
Comment (wizardlink) [inline src/structures/Client.ts]: Why comment and not straight up remove?

**Review by wizardlink — APPROVED** [review]

**Pull request merged**